src/features/hostapp/hooks/index.ts

```diff
@@ -0,0 +1 @@
+import './target-hostapp';
```

Comment:
You can remove this file altogether since the hooks are imported directly not via this index